### ISSUE TITLE
Highlighter - AutoToggle is active during function highlighter.highlightByID

### ISSUE THREAD

**Issue by TobiasSeum** [ThatOpen/engine_components#463]
### Describe the bug 📝

Using the new feature AutoToggle in in combination with the method highlightByID(...)

Lets say I have an element x.

if I have the element x in the current selection (it is marked), the interaction with "mouse events" work fine and intuitive.
But if I have a make a function call highlightById(...) to make sure element x is marked, then the autotoggle deselect my element.

This is for my understanding a bug. Analysing the current selection is not a good option.

Without knowing details of the original concept I would suggest 2 things:
1: trigger autotoggle only for mouse interaction (raycaster)
2: having another parameter in the highlightByID (select or deselect) |or| providing a function to deselect elements by id 

### System Info 💻

```shell
"@thatopen/components": "^2.1.20",
        "@thatopen/components-front": "^2.1.17",
        "@thatopen/fragments": "^2.1.6",
        "@thatopen/ui": "^2.1.3",
        "@thatopen/ui-obc": "^2.1.5",
```

### Used Package Manager 📦

npm

### Validations ✅

- [X] Read the [docs](https://docs.thatopen.com/intro).
- [X] Check that there isn't [already an issue](https://github.com/ThatOpen/engine_components/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a repository issue and not a framework-specific issue. For example, if it's a THREE.js related bug, it should likely be reported to [mrdoob/threejs](https://github.com/mrdoob/three.js) instead.
- [X] Check that this is a concrete bug. For Q&A join our [Community](https://people.thatopen.com/).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by agviegas** [comment]
@TobiasSeum I've added a new optional parameter to `highlightByID` which allows to specify if it's a picking operation with the mouse. The autotoggle will only work if that's true. That way, your problem will be solved. Available from `@thatopen/components-front@2.1.19`.

**Comment by TobiasSeum** [comment]
Wow, this was crazy fast! Thank you!
